siliconcompiler/report/dashboard/web/__init__.py

```
@@ -46,7 +46,7 @@ def __init__(self, project, port=None, graph_projs=None):
         try:
             from streamlit.web import bootstrap  # noqa: F401
         except ModuleNotFoundError:
-            raise NotImplementedError('streamlit is not available for dashboard')
+            raise ModuleNotFoundError('streamlit is not available for dashboard')
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)